src/client/cli/formatter/csv_formatter.cpp

```diff
@@ -46,7 +46,35 @@ std::string format_images(const google::protobuf::RepeatedPtrField<mp::FindReply
     return fmt::to_string(buf);
 }
 
-std::string generate_instance_info_report(const mp::InfoReply& reply)
+std::string generate_snapshot_details(const mp::InfoReply reply)
+{
+    fmt::memory_buffer buf;
+
+    fmt::format_to(std::back_inserter(buf),
+                   "Snapshot,Instance,Size,CPU(s),Disk space,Memory size,Mounts,Created,Parent,Children,Comment\n");
```

Comment:
Only showing it when available wouldn't work with csv because of column positions. I've removed it since it isn't implemented. Does the same need to be done for yaml and json then, too?

Reply:
Yes, I don't think we should include the size if it is not available, regardless of the format. Good point on omission not working with CSV. When we implement it, we'll need to add "N/A" or "-" in backends that do not make it available.